src/main/webapp/app/pages/apiAccessGroup/APIAccessPage.tsx

```diff
@@ -38,48 +38,92 @@ const getDataTitle = (date: string, version: string) => {
   return `${getNewsTitle(date)} (${version})`;
 };
 
+function getMajorVersion(versionString: string): number | undefined {
+  const match = versionString.match(/^v(\d+)\./);
+
+  return match ? parseInt(match[1], 10) : undefined;
+}
+function getMinorVersion(versionString: string): number | undefined {
+  const match = versionString.match(/^v(\d+)\.(\d+)/);
+
+  return match ? parseInt(match[2], 10) : undefined;
+}
+
 const BUTTON_CLASS_NAME = 'mr-2 my-1';
 const DownloadButtonGroups: React.FunctionComponent<{
   data: DownloadAvailabilityWithDate;
 }> = props => {
+  const majorVersion = getMajorVersion(props.data.version) ?? 0;
+  const minorVersion = getMinorVersion(props.data.version) ?? 0;
+  const versionIsLessThan4 = majorVersion < 4;
   return (
     <>
       {props.data.hasAllCuratedGenes ? (
         <AuthDownloadButton
           className={BUTTON_CLASS_NAME}
           fileName={`all_curated_genes_${props.data.version}.tsv`}
-          getDownloadData={() =>
-            oncokbClient.utilsAllCuratedGenesTxtGetUsingGET({
+          getDownloadData={() => {
+            return oncokbClient.utilsAllCuratedGenesTxtGetUsingGET({
               version: props.data.version,
-            })
-          }
+            });
+          }}
           buttonText="All Curated Genes"
         />
       ) : null}
       {props.data.hasCancerGeneList ? (
         <AuthDownloadButton
           className={BUTTON_CLASS_NAME}
           fileName={`cancer_gene_list_${props.data.version}.tsv`}
-          getDownloadData={() =>
-            oncokbClient.utilsCancerGeneListTxtGetUsingGET({
+          getDownloadData={() => {
+            return oncokbClient.utilsCancerGeneListTxtGetUsingGET({
               version: props.data.version,
-            })
-          }
+            });
+          }}
           buttonText="Cancer Gene List"
         />
       ) : null}
       {props.data.hasAllActionableVariants ? (
-        <AuthDownloadButton
-          className={BUTTON_CLASS_NAME}
-          fileName={`oncokb_${props.data.version.replace('.', '_')}.sql.gz`}
-          getDownloadData={async () => {
-            const data = await oncokbPrivateClient.utilDataSqlDumpGetUsingGET({
-              version: props.data.version,
-            });
-            return data;
-          }}
-          buttonText="Data Dump"
-        />
+        <>
+          <AuthDownloadButton
+            className={BUTTON_CLASS_NAME}
+            fileName={`oncokb_${props.data.version.replace('.', '_')}.sql.gz`}
+            getDownloadData={async () => {
+              const data = await oncokbPrivateClient.utilDataSqlDumpGetUsingGET(
+                {
+                  version: props.data.version,
+                }
+              );
+              return data;
+            }}
+            buttonText="Data Dump"
+          />
+          <AuthDownloadButton
```

Comment:
Might be a good idea add a small info icon or some way to notify the user what the transcript data is.
`The transcript database serves OncoKB metadata included gene, transcript, sequence, etc. Only required for local installations that will utilize the /byGenomicChange and /byHGVSg endpoints`